generated/nifgen/session.py

```diff
@@ -2222,6 +2222,41 @@ def _lock_session(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=True)
         return
 
+    @ivi_synchronized
+    def send_software_edge_trigger(self):
+        '''send_software_edge_trigger
+
+        Sends a command to trigger the signal generator. This VI can act as an
+        override for an external edge trigger.
+
+        If called directly on the session, this will send a software start trigger.
+
+            session.send_software_edge_trigger()
+
+        If called using the script trigger repeated capability container, this will
+        send a software trigger to the specified script trigger
+
+            session.script_triggers[1].send_software_edge_trigger()
+
+        Note:
+        This method does not override external digital edge triggers of the
+        NI 5401/5411/5431.
+        '''
+        # We look at whether we are called directly on the session or a repeated capability container to determine how to behave
+        if len(self._repeated_capability) > 0:
+            trigger_id = self._repeated_capability
+            trigger = enums.Trigger.SCRIPT
```

Comment:
I think it would be reasonable to hardcode the values in this function plus a comment.

Reply:
Changed